tests/Unit/Evolution/VariableFixing/Test_LimitLorentzFactor.cpp

```diff
@@ -0,0 +1,90 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <array>
+#include <cmath>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/DotProduct.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/VariableFixing/LimitLorentzFactor.hpp"
+#include "PointwiseFunctions/Hydro/LorentzFactor.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "tests/Unit/TestCreation.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+// IWYU pragma: no_forward_declare Tensor
+
+namespace VariableFixing {
+namespace {
+void test_variable_fixer(const LimitLorentzFactor& variable_fixer) {
+  const Scalar<DataVector> density{DataVector{1., 2., 1.e-5, 1e-6}};
```

Comment:
unconsistent number formatting
use e.g. `1e-5`, but make sure you are consistent